src/components/todoFooter.tsx

```diff
@@ -0,0 +1,72 @@
+import { Dispatch, SetStateAction, useState } from 'react';
+import classNames from 'classnames';
+import { FilterBy } from '../types/FilterBy';
+
+interface Props {
+  setFilterBy: Dispatch<SetStateAction<FilterBy>>,
+  itemsQuantity: number,
+}
+
+export const TodoFooter: React.FC<Props> = ({ setFilterBy, itemsQuantity }) => {
+  const navItems = [
+    {
+      title: 'All',
+      id: 'ALL',
```

Comment:
please use `enum FilterBy` to avoid issues
```suggestion
      id: FilterBy.ALL,
```